src/app/Scenes/Consignments/Screens/SubmitArtworkOverview/Mutations/createConsignSubmissionMutation.ts

```diff
@@ -19,11 +19,7 @@ export const createConsignSubmission = (input: CreateSubmissionMutationInput) =>
         }
       `,
       variables: {
-        input: {
-          ...input,
-          source: "APP_INBOUND",
-          userAgent: getCurrentEmissionState().userAgent,
-        },
+        input: { ...input, userAgent: getCurrentEmissionState().userAgent },
```

Comment:
Can we do this instead of adding a migration?

```suggestion
        input: {
          ...input,
          source: input.source ?? "APP_INBOUND",
          userAgent: getCurrentEmissionState().userAgent,
        },
```